### PR TITLE
running an ONNX model exported from PyTorch pretrained model

### PR DESCRIPTION
**Pull request by QiJune**
*Description*: running an ONNX model exported from PyTorch pretrained model
*Testing*:
*Documentation*:

### CODE REVIEW

**Comment by nadavrot** [comment]
@QiJune Thank you for the pull request. I don't understand this topic, but I think that it's reasonable to have an example that shows how to export models from PyTorch, so I think that we should approve this PR. Could you add a short document that explains what this example is and how to use it?

File: examples/pytorch/Makefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# The path to the image-classifier executable.
+LOADER?=~/src/build/glow/bin/image-classifier
```
[the remark this answers is not in the document]
Comment (jfix71) [reply examples/pytorch/Makefile]: I believe these were taken from the example bundles Makefiles. They are just the default values in case an env var with the same name is not set -- this is what `?=` does. (Same on `GLOW_SRC` below too)

**Comment by rdzhabarov** [comment]
> I'm not sure the Makefile brings adds a lot of value here. I think it may make more sense to just put export_onnx_model.py in utils/, and then add better descriptions to the parser arguments. We already have a couple scripts there for downloading models from AWS.

+1 on this

File: examples/pytorch/Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (QiJune) [reply examples/pytorch/Makefile]: @artemrakhov @jfix71 Yes, I just copy the env var from the example bundles Makefiles.

And I have tested the command in my server:

```bash
qijun@junq-5520:/mnt/c/Users/junq/works/glow/examples/pytorch$ LOADER=/mnt/c/Users/junq/works/glow/build/bin/image-classifier GLOW_SRC=/mnt/c/Users/junq/works/glow make run
python export_onnx_model.py --input_name data --model_name resnet18 --model_path resnet18.onnx
/mnt/c/Users/junq/works/glow/build/bin/image-classifier /mnt/c/Users/junq/works/glow/tests/images/imagenet/cat_285.png -image_mode=0to1 -m resnet18.onnx -model_input_name=data -cpu
Model: resnet18.onnx
 File: /mnt/c/Users/junq/works/glow/tests/images/imagenet/cat_285.png   Label-K1: 285 (probability: 16.4305)
```

The result is good

**Comment by QiJune** [comment]
@nadavrot @rdzhabarov @jfix71 @artemrakhov 
As discussed, I will move `export_onnx_model.py`  to `utils\` directory, and write a simple markdown doc to provide more usage details.

File: docs/ModelLoader.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+# Please keep the model_input_name same with export_onnx_model util function
+${GLOW_PATH}/build/bin/image-classifier ${GLOW_PATH}/tests/images/imagenet/cat_285.png -image_mode=0to1 -m resnet18.onnx -model_input_name=data -cpu
+```
+
+
+The running results can be:
+
+```bash
+Model: resnet18.onnx
+ File: ${GLOW_PATH}/tests/images/imagenet/cat_285.png   Label-K1: 285 (probability: 16.4305)
```
Comment (rdzhabarov) [inline docs/ModelLoader.md]: hm, the probability is > 1, is it because there is no softmax layer there?

From the table above for top1 error:
> |resnet18                    |     30.24

File: docs/ModelLoader.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+cd ${GLOW_PATH}/utils
+# This command export resnet18 model from PyTorch, and save the model as resnet18.onnx in current directory
+python export_onnx_model.py --model_name resnet18 --model_path resnet18.onnx --model_input_name data
+
+# Export alexnet model
+python export_onnx_model.py --model_name alexnet --model_path alexnet.onnx --model_input_name data
+```
+
+
+Afther we get the ONNX model, we can load it and run it in Glow.
```
Comment (rdzhabarov) [inline docs/ModelLoader.md]: typo

File: docs/ModelLoader.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+## Model Loader
+
+This document describes how to load a pretrained model  from other deep leraning framework, such as PyTorch and Caffe2, to Glow and then execute inside.
+
+
```
Comment (rdzhabarov) [inline docs/ModelLoader.md]: can you remove empty lines here

File: docs/ModelLoader.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
+## Model Loader
+
+This document describes how to load a pretrained model  from other deep leraning framework, such as PyTorch and Caffe2, to Glow and then execute inside.
```
Comment (rdzhabarov) [inline docs/ModelLoader.md]: are there any reference to Caffe2 in this doc?

Comment (nadavrot) [reply docs/ModelLoader.md]: double space between "model  from"

**Comment by nadavrot** [comment]
@QiJune Thank you for doing this work.

File: docs/ModelLoader.md (hunk 5 above)
Comment (jfix71) [inline docs/ModelLoader.md]: typo: "leraning"

File: utils/export_onnx_model.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
+from torch.autograd import Variable
```
Comment (jfix71) [inline utils/export_onnx_model.py]: I believe we need to add the Copyright header here. Can you copy from `utils/download_test_db.py`?

**Comment by artemrakhov** [comment]
LGTM. Feel free to merge after addressing other comments.

File: docs/ModelLoader.md (hunk 3 above)
Comment (QiJune) [reply docs/ModelLoader.md]: Done

File: docs/ModelLoader.md (hunk 4 above)
Comment (QiJune) [reply docs/ModelLoader.md]: Done

File: docs/ModelLoader.md (hunk 5 above)
Comment (QiJune) [reply docs/ModelLoader.md]: Will add Caffe2 part in next PR

Comment (QiJune) [reply docs/ModelLoader.md]: Done

File: utils/export_onnx_model.py (hunk 6 above)
Comment (QiJune) [reply utils/export_onnx_model.py]: Done

**Comment by nadavrot** [comment]
Can we lang this PR?
